src/test/resources/fixtures/bulkscan/transformation/simple-formA.json

```diff
@@ -2,7 +2,7 @@
   "case_type_id": "FinancialRemedyContested",
   "id": "LV481297",
   "po_box": "PO 17",
-  "form_type": "formA",
+  "form_type": "FormA",
```

Comment:
why don't we also rename a file to `simple-FormA.json`?

Reply:
done